paasta_tools/apply_external_resources.py

```diff
@@ -0,0 +1,76 @@
+#!/opt/venvs/paasta-tools/bin/python
+import os
+import sys
+from filecmp import cmp
+from shutil import copy
+from subprocess import CalledProcessError
+from subprocess import run
+from traceback import print_exc
+
+APPLIED_DIRECTORY = ".applied"
+
+
+# This script expects the KUBECONFIG environment variable to be set correctly
+def main(puppet_resource_root: str) -> int:
+    exit_code = 0
+    applied_resource_root = os.path.join(puppet_resource_root, APPLIED_DIRECTORY)
```

Comment:
Did we fix the issue where puppet might purge the `.applied` directory?

Reply:
I'm going to have Puppet create and manage the `.applied` directory.